peekingduck/pipeline/nodes/model/yolov4/yolo_files/detector.py

```diff
@@ -186,7 +187,7 @@ def predict_object_bbox_from_image(
         )
 
         # convert classes into class names
-        classes = np.array([class_names[int(i)] for i in classes])  # type: ignore
+        classes = np.array([self.class_names[int(i)] for i in classes])  # type: ignore
 
         return boxes, classes, scores  # type: ignore
```

Comment:
Might be a good chance to make the detector classes more consistent by changing this to be
```
labels = ...

return boxes, labels, scores
```
instead? I think this lets us remove the type ignore magic comments for those 2 lines

Reply:
Used new `labels` var as proposed.